src/containers/Full/Full.js

```diff
@@ -93,7 +93,15 @@ class Full extends Component {
           active: isActive('/validation', pathname),
           primaryText: 'Validation',
           leftIcon: <FontIcon>insert_chart</FontIcon>,
-        }/* ,
+        },
+        {
+          component: Link,
+          to: '/explanation',
+          active: isActive('/explanation', pathname),
+          primaryText: 'explanation',
+          leftIcon: <FontIcon>work</FontIcon>,
+        },
+        /* ,
```

Comment:
as the other, please comment this menu item so that only we are capable to reach the page until it is ready.